### DISCUSSION TITLE
Cantaloupe Start

### DISCUSSION THREAD

**Discussion by Chillo97** [cantaloupe-project/cantaloupe#645] [General]
can someone please help me. I don't know why I can't start Cantaloupe on mine.
java -Dcantaloupe.config=C:/cantaloupe/cantaloupe.properties -Xmx2g -jar cantaloupe-5.0.6.jar   (That does not work)

**Comment by glenrobson** [comment]
Hi @Chillo97, could you send what was printed after you ran that command and I can try and see what is going on?

Thanks Glen

**Reply by Chillo97** [reply]
Hallo @glenrobson,
Thank you very much for wanting to help me. There is no error message. I made a little video and saved it in Drive, there you can see what I did and maybe why it doesn't work.

https://drive.google.com/file/d/1okT-5rD4aNv3zsTx7IzdZyP2jh-xx5u5/view?usp=sharing

Achille

**Comment by glenrobson** [comment]
Thank you that video was very helpful. Its strange you didn't get any error or output from the Cantaloupe command. Have you got Java installed and if so do you know which version? Thanks

**Reply by Chillo97** [reply]
Thanks for your help, it works, but I would like to ask you something again. Have you already integrated UniversalViewer or Mirador into a React project? If so, can you show me how you did that? Thanks in advance.

**Reply by glenrobson** [reply]
I'm afraid I haven't but I would ask on the respective slacks to see if someone can help. Mirador uses a chanel on the IIIF slack which you can register at:

http://bit.ly/iiif-slack

and the Universal Viewer has its own slack which you can register at:

https://docs.google.com/forms/d/e/1FAIpQLSeHLD0kng5aXvGFsNN_tJGsZMTnp08Hv2F6kdGsJRb6bT0NWw/viewform

**Reply by Chillo97** [reply]
Thank you very much
